src/OSPSuite.Core/Domain/DistributedParameter.cs

```diff
@@ -24,6 +25,7 @@ public class DistributedParameter : QuantityAndContainer, IDistributedParameter
       public ParameterInfo Info { get; set; }
       public ParameterOrigin Origin { get; private set; }
       public double? DefaultValue { get; set; }
+      public DescriptorCriteria ContainerCriteria { get; set; }
```

Comment:
I have added this property at the parameter level. Null by default for all parameter so that we only create it when needed